src/scss/styles.scss

```diff
@@ -1,20 +1,51 @@
 @import "bourbon";
 @import "neat";
 
+@import url(http://fonts.googleapis.com/css?family=Raleway:400,600,700,300);
+@import url(http://fonts.googleapis.com/css?family=Roboto:400,600,700,300);
+

```

Comment:
I agree with. It was mainly just testing the reload after changing fonts + saving.